pkg/apis/servicecatalog/serialization_test.go

```diff
@@ -28,10 +28,10 @@ import (
 	proto "github.com/golang/protobuf/proto"
 	flag "github.com/spf13/pflag"
 
+	apitesting "github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog/testing"
```

Comment:
I don't understand what you guys are saying here. If you're annotating what that import is for, you got it right. If you're trying to tell me something, it went over my head.